### DISCUSSION TITLE
Binary patch for basic functionality?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [Eloston/ungoogled-chromium#1451] [Ideas]
Would it be feasible to patch out the built-in google URLs with patches to the chromium binary? This way the user would not be required to use "unofficial" builds (non-Google/non-package manager) or build ungoogled-chromium themselves to get the basic google request blocking functionality.

**Comment by megastallman** [comment]
You need to patch both ELFs and *.pak(actually zipped resource files). I've tried, it last year, but failed. That was not Chrome/Chromium, but Opera, which is pretty much the same, but with a free proxy service and much more junk.
If you figure out how to avoid breaking binary offsets, then probably you have some kind of success, but it is definitely not a replacement to Ungoogled.
